.github/ISSUE_TEMPLATE/all-for-one.yml

```
@@ -0,0 +1,90 @@
+name: All for One, One For All bounty submission
+description: Submit a CodeQL query for the All For One, One For All bounty (https://securitylab.github.com/bounties#allforone)
+title: "[<language>]: <short description>"
+labels: [All For One]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        # Introduction
+
+        Thank you for submitting a query to the GitHub CodeQL project!
+
+        After you submit this issue, the GitHub Security Lab and CodeQL teams will triage the submission and, if it meets the Query Bounty Program requirements, we will grant you a bounty through our HackerOne program.
+
+        Please make sure to carefully read the [bounty program description and conditions](https://securitylab.github.com/bounties#allforone)
+
+        # Questionnaire
+  - type: input
+    id: pr_url 
+    attributes:
+      label: Query PR
+      description: Link to pull request with your CodeQL query
+      placeholder: |
+        ex. https://github.com/github/codeql/pull/nnnn
+    validations:
+      required: true
+  - type: dropdown
+    id: language
+    attributes:
+      label: Language
+      description: What programming language is your query written for?
+      options:
+        - Java
+        - Javascript
+        - GoLang
+        - Python
+        - C/C++
+        - C#
+    validations:
+      required: true
+  - type: textarea
+    id: cve_ids
+    attributes:
+      label: CVE(s) ID list
+      description: Enter a list of the CVE ID(s) associated with this query, one bullet for each distinct CVE. GitHub will automatically link CVE IDs to the [GitHub Advisory Database](https://github.com/advisories). If the result(s) is **NOT YET** fixed **nor disclosed**, and you are still waiting for a CVE, then you can privately share your result via email to [security@github.com](mailto:security@github.com?subject=[BugBounty]%20Issue%20#000%20useful%20result)
+      placeholder: |
+        ex.
+        - [CVE-20nn-xxxx](<relevant URL>)
+        - [CVE-20nn-yyyy](<relevant URL>)
+    validations:
+      required: true
+  - type: input
+    id: cwe
+    attributes:
+      label: CWE
+      description: [CWE](https://cwe.mitre.org/data/index.html) that best fits the vulnerability class modeled with your query
```

```suggestion
      description: "[CWE](https://cwe.mitre.org/data/index.html) that best fits the vulnerability class modeled with your query"
```
Enclosing the description in quotes seems to fix this error:
> There is a problem with this template
>
> YAML error: (): did not find expected key while parsing a block mapping at line 55 column 7

Otherwise the YAML parser seems to think that `[CWE]` or `label: CWE` (line above) refers to some kind of mapping, I guess.